build/Nuget/Sdkproject.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!--
+       This project is being used as a wrapper project for packing with the nuspec.
+       https://github.com/NuGet/Home/wiki/Adding-nuget-pack-as-a-msbuild-target#packing-using-a-nuspec
+  -->
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <NuspecFile>Microsoft.NET.Sdk.nuspec</NuspecFile>
+  </PropertyGroup>
+
+  <Target Name="SetPackageDependencies" BeforeTargets="GenerateNuspec" >
+    <PropertyGroup>
+       <NuspecBasePath>..\..\bin\$(Configuration)</NuspecBasePath>
+       <NuspecProperties>version=$(Version)</NuspecProperties>
```

Comment:
Is the version coming out the same as before. I think common.props is not auto imported in this directory.